src/app/services/xml-parser-service/xmp-parser.service.ts

```diff
@@ -102,8 +102,14 @@ export const ContentParser = (content: any): string => {
   } else if (
     content instanceof org.cru.godtools.shared.tool.parser.model.Accordion
   ) {
-    // console.log('CONTENT: Accordion');
+    console.log('CONTENT: Accordion');
     return 'accordion';
+  } else if (
+    content instanceof
+    org.cru.godtools.shared.tool.parser.model.Accordion.Section
+  ) {
+    console.log('CONTENT: Section');
+    return 'section';
```

Comment:
I don't know if you need to include this here, an `Accordion.Section` doesn't extend the `Content` interface, so it should never show up in a `Content` array. It should only be used in the `accordion.sections` array.

Reply:
I've removed this type